bike-erp/backend/src/main/services/accountingService/AccountingService.ts

```diff
@@ -57,7 +58,7 @@ export class AccountingService {
           transactionItemId
         );
       });
-
+      UserLogService.addLog(email, "Created Account Payable").catch((error)=> {});
```

Comment:
if we're not handling the error, I think we should just omit the catch part.

Reply:
Travis requires me to handle it, I can add an error message? I dont think much else will be useful

Reply:
Oh I see, no need to change then